### ISSUE TITLE
404 not found zlib-1.2.11.tar.gz from www.zlib.net.  Fallback "pocketcoincore.org" domain does not exist.

### ISSUE THREAD

**Issue by jkaldon** [pocketnetteam/pocketnet.core#263]
**Describe the bug**
The archive zlib-1.2.11.tar.gz is no longer hosted at www.zlib.net.  The fallback domain is also invalid.  

**To Reproduce**
1. git clone --depth 1 --branch v0.20.19 https://github.com/pocketnetteam/pocketnet.core.git
2. cd pocketnet.core/depends
3. make download-linux

**Expected behavior**
zlib archive should be downloaded.

**Screenshots**
<img width="1280" alt="Screen Shot 2022-04-09 at 7 26 08 PM" src="https://user-images.githubusercontent.com/1554651/162596042-34b31f52-c986-41a6-947f-2ecf2024c33a.png">

**Desktop (please complete the following information):**
 - OS: Debian
 - Browser Not Applicable
 - Version Buster

**Smartphone (please complete the following information):**
 - Device: Not Applicable
 - OS: Not Applicable
 - Browser Not Applicable
 - Version Not Applicable

**Additional context**
None

**Comment by jkaldon** [comment]
After updating `depends/packages/zlib.mk` with the following version and hash, I was able to build successfully.

```
$(package)_version=1.2.12
...
$(package)_sha256_hash=91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9
```

Let me know and I can submit a patch, if this is the preferred fix.

**Comment by andyoknen** [comment]
@jkaldon It would be very nice of you - we will accept your PR in one of the next releases.

**Comment by jkaldon** [comment]
> @jkaldon It would be very nice of you - we will accept your PR in one of the next releases.

@andyoknen Sorry, I just realized that I need some guidance on things like "Which branch should I create my branch from?"  "What should I name my branch?", etc.

I'm want to submit other PRs in the future, so this is a learning investment I should make.  I reviewed the content of `doc/developer-notes.md`, but it references a master branch...which doesn't exist.  I see the default branch is `0.20`, should I branch off of that or branch of `development` or something else?
